plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -210,21 +210,24 @@ class ServicesTable extends React.Component {
     ];
 
     return (
-      <Dropdown
-        key="actions-dropdown"
-        anchorRight={true}
-        buttonClassName="button button-mini dropdown-toggle button-link table-display-on-row-hover"
-        dropdownMenuClassName="dropdown-menu"
-        dropdownMenuListClassName="dropdown-menu-list"
-        dropdownMenuListItemClassName="clickable"
-        wrapperClassName="dropdown flush-bottom table-cell-icon"
-        items={dropdownItems}
-        persistentID={ServiceActionItem.MORE}
-        onItemSelection={this.onActionsItemSelection.bind(this, service)}
-        scrollContainer=".gm-scroll-view"
-        scrollContainerParentSelector=".gm-prevented"
-        transition={true}
-        transitionName="dropdown-menu" />
+      <Tooltip interactive={true} content={'More actions'}>
```

Comment:
Let's adjust this a little bit to: `<Tooltip content="More actions">`

We don't want `interactive={true}` in this case (the component defaults to `interactive={false}`. The `interactive` property toggles the ability for the user's cursor to enter the tooltip vs being dismissed when the user's cursor leaves the trigger element. We set `interactive={true}` when the tooltip contains links or actions that the user needs to interact with.